nodejs/week4/homework-template/src/frontend/pages/home.js

```diff
@@ -0,0 +1,59 @@
+const searchSpecificMeal = event => {
+	const searchedTitle = this.event.target.value;
+	fetch(`/api/meals?title=${searchedTitle}`)
+		.then(response => response.json())
+		.then(meals => renderMeals(meals))
+		.catch(err => console.log("i am searchspecific meals error", err));
+};
+
+const fetchAllMeals = () => {
+	fetch('/api/meals')
+		.then(response => response.json())
+		.then(meals => renderMeals(meals))
+		.catch(err => console.log("i am fetch all meals error", err));
+};
+
+const renderMeals = meals => {
+  document.getElementById('meals').innerHTML = meals
+    .filter((meal) => meal.Title)
```

Comment:
It would be better to ensure that meals always have a title when creating it (in the backend route).